src/webgpu/api/validation/copyBetweenLinearDataAndTexture_writeTextureSpecific.spec.ts

```diff
@@ -0,0 +1,126 @@
+export const description = `
+writeTexture validation tests.
+`;
+
+import { params, poptions, pbool } from '../../../common/framework/params_builder.js';
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { kTextureFormats, kTextureFormatInfo } from '../../capability_info.js';
+
+import { CopyBetweenLinearDataAndTextureTest, TestMethod } from './copyBetweenLinearDataAndTexture.js';
+
+export const g = makeTestGroup(CopyBetweenLinearDataAndTextureTest);
+
+g.test('bound_on_bytes_per_row')
```

Comment:
Maybe these two tests could be factored between the copies and writeTexture, with additional parameters, and a check for the bytesPerRow to be a multiple of 256 in the copy cases. (with an unless combinator)